src/deploy.js

```diff
@@ -0,0 +1,16 @@
+import { spawn } from './utils';
+
+const deploy = async (opts) => {
+  const { path } = opts;
+
+  console.error(`path passed '${path}'`);
+  console.log(process.cwd());
+  try {
+    await spawn('./src/test.sh');
```

Comment:
This should be `./test.sh` after compiling inside the `lib` folder.

Reply:
@juancarlosfarah the test.sh file is not copied into the lib folder. I suppose babel is only considering `.js` file during compilation.
But even if I move the file manually, the command is executed at the location of the terminal and the path interpreted as relative path.